Global/mise.gitignore

```diff
@@ -0,0 +1,17 @@
+# https://mise.jdx.dev/configuration.html
+# https://mise.jdx.dev/configuration/environments.html
+/.mise.*.local.toml
+/.mise.local.toml
+/mise.*.local.toml
+/mise.local.toml
+/.mise/config.local.toml
+/.mise/config.*.local.toml
+/mise/config.local.toml
+/mise/config.*.local.toml
+/.config/mise.local.toml
+/.config/mise.*.local.toml
+/.config/mise/config.local.toml
+/.config/mise/config.*.local.toml
```

Comment:
A simplified version that might work better with e.g. monorepos (i.e. setups where there are separate mise config sets in subdirs) could be:
```
.mise.*.local.toml
.mise.local.toml
mise.*.local.toml
mise.local.toml
.mise/*.local.toml
mise/*.local.toml
```

Reply:
Done that.